salt/states/svn.py

```diff
@@ -84,6 +85,12 @@ def latest(name,
 
     trust : False
         Automatically trust the remote server. SVN's --trust-server-cert
+
+    trust_failures : None
+        Comma-separated list of certificate trust failures, that shall be
+        ignored. This can be used if trust=True is not sufficient. The
+        specified string is passed to SVN's --trust-server-cert-failures
+        option as-is.
```

Comment:
Can you add a `.. versionadded:: Fluorine` tag to this new option?

Reply:
Sure, I just pushed the changes.